### DISCUSSION TITLE
Superuser and LDAP

### DISCUSSION THREAD

**Discussion by dkoci** [quay/quay#990] [Q&A]
Hi.
I am a new quay user. My first configuration was default one (local database for internal access with some superuser). Then I changed it to use LDAP. Now I can login with my LDAP credentials but I cant log in with previously created superuser anymore.
Is it possible to have some superuser in local database and other "normal" users in LDAP?

Another question: is there any option to use LDAPS instead of LDAP (there is no 'add certificates' option for ldaps)?

Tnx

**Comment by dmesser** [comment]
Hi, thanks for trying Project Quay. LDAP is considered an exclusive authentication source, so it's either LDAP or the local database defining users, including superusers. LDAPS is supported, simply use "ldaps://" in the URI. Certificates will be taken from the extra certs portion of the Quay configuration.
